NSS-Day-Cohort-49/giffygram-success-kids
# Lam gif

#### Changes Made
1. Added file `NavBar.js and login.css` to `Nav and styles` directory.
1. Modified file `PostList.js` to include `html changes to posts being displayed` .
2. Added styling to login page and navbar 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user clicks on logout button, then user should be logged out of main page and redirected to the login page. 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
